<ui version="4.0" >
 <class>PgmArgsBase</class>
 <widget class="QDialog" name="PgmArgsBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Program Arguments</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="argsPage" >
        <attribute name="title" >
         <string>&amp;Arguments</string>
        </attribute>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="labelArgs" >
             <property name="text" >
              <string>Run &lt;i>%1&lt;/i> with these arguments:</string>
             </property>
             <property name="wordWrap" >
              <bool>false</bool>
             </property>
             <property name="buddy" >
              <cstring>programArgs</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="programArgs" >
             <property name="whatsThis" >
              <string>Specify the arguments with which the program shall be invoked for this debugging session. You specify the arguments just as you would on the command line, that is, you can even use quotes and environment variables, for example:&lt;p>&lt;tt>--message 'start in: ' $HOME&lt;/tt></string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QPushButton" name="insertFile" >
               <property name="whatsThis" >
                <string>Browse for a file; the full path name will be inserted at the current cursor location in the edit box above.</string>
               </property>
               <property name="text" >
                <string>Insert &amp;file name...</string>
               </property>
               <property name="shortcut" >
                <string>Alt+F</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="insertDir" >
               <property name="whatsThis" >
                <string>Browse for a directory; the full path name will be inserted at the current cursor location in the edit box above.</string>
               </property>
               <property name="text" >
                <string>Insert &amp;directory name...</string>
               </property>
               <property name="shortcut" >
                <string>Alt+D</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacer1" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>61</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="spacer2" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>81</width>
               <height>180</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="wdPage" >
        <attribute name="title" >
         <string>&amp;Working Directory</string>
        </attribute>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLineEdit" name="wdEdit" >
             <property name="whatsThis" >
              <string>Specify here the initial working directory where the program is run.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QPushButton" name="wdBrowse" >
               <property name="whatsThis" >
                <string>Browse for the initial working directory where the program is run.</string>
               </property>
               <property name="text" >
                <string>&amp;Browse...</string>
               </property>
               <property name="shortcut" >
                <string>Alt+B</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacer4" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>321</width>
                 <height>31</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="spacer5" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>111</width>
               <height>161</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="envPage" >
        <attribute name="title" >
         <string>&amp;Environment</string>
        </attribute>
        <layout class="QHBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QLabel" name="envLabel" >
               <property name="text" >
                <string>Environment variables (&lt;tt>NAME=value&lt;/tt>):</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
               <property name="buddy" >
                <cstring>envVar</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="envVar" >
               <property name="whatsThis" >
                <string>To add a new environment variable or to modify one, specify it here in the form &lt;tt>NAME=value&lt;/tt> and click &lt;b>Modify&lt;/b>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Q3ListView" name="envList" >
               <property name="whatsThis" >
                <string>Environment variables that are set &lt;i>in addition&lt;/i> to those that are inherited are listed in this table. To add new environment variables, specify them as &lt;tt>NAME=value&lt;/tt> in the edit box above and click &lt;b>Modify&lt;/b>. To modify a value, select it in this list and click &lt;b>Modify&lt;/b>. To delete an environment variable, select it in this list and click &lt;b>Delete&lt;/b>.</string>
               </property>
               <property name="selectionMode" >
                <enum>Q3ListView::Single</enum>
               </property>
               <column>
                <property name="text" >
                 <string>Name</string>
                </property>
                <property name="clickable" >
                 <bool>false</bool>
                </property>
                <property name="resizable" >
                 <bool>true</bool>
                </property>
               </column>
               <column>
                <property name="text" >
                 <string>Value</string>
                </property>
                <property name="clickable" >
                 <bool>false</bool>
                </property>
                <property name="resizable" >
                 <bool>true</bool>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QPushButton" name="buttonModify" >
               <property name="whatsThis" >
                <string>Enters the environment variable that is currently specified in the edit box into the list. If the named variable is already in the list, it receives a new value; otherwise, a new entry is created.</string>
               </property>
               <property name="text" >
                <string>&amp;Modify</string>
               </property>
               <property name="shortcut" >
                <string>Alt+M</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonDelete" >
               <property name="whatsThis" >
                <string>Deletes the selected environment variable from the list. This cannot be used to delete environment variables that are inherited.</string>
               </property>
               <property name="text" >
                <string>&amp;Delete</string>
               </property>
               <property name="shortcut" >
                <string>Alt+D</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacer6" >
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>51</width>
                 <height>141</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="xsldbgOptionsPage" >
        <attribute name="title" >
         <string>&amp;xsldbg Options</string>
        </attribute>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="Q3ListBox" name="xsldbgOptions" >
           <property name="selectionMode" >
            <enum>Q3ListBox::Multi</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer name="spacer3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOk" >
         <property name="text" >
          <string>&amp;OK</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel" >
         <property name="text" >
          <string>&amp;Cancel</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>programArgs</tabstop>
  <tabstop>insertFile</tabstop>
  <tabstop>insertDir</tabstop>
  <tabstop>wdEdit</tabstop>
  <tabstop>wdBrowse</tabstop>
  <tabstop>envVar</tabstop>
  <tabstop>buttonModify</tabstop>
  <tabstop>buttonDelete</tabstop>
  <tabstop>envList</tabstop>
  <tabstop>xsldbgOptions</tabstop>
  <tabstop>buttonOk</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>PgmArgsBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>409</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>PgmArgsBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>464</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
